rosidl_generator_py/resource/_msg.py.em

```diff
@@ -18,6 +19,52 @@ from rosidl_parser.definition import Sequence
 from rosidl_parser.definition import String
 from rosidl_parser.definition import WString
 }@
+@#<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
```

Comment:
Would you consider using a different delimiter here?  These look a lot like git conflict markers.

Reply:
This kind of delimiter already exists in numerous em templates throughout the various generators. So for this PR I would rather stick with it. But a set of follow up PRs could replace them with whatever else is preferred.

Reply:
Ah, sorry, I guess I hadn't noticed it before.  It really jumped out at me this time.  I'm good with it staying, then.